app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id="group">
+  <div class= "image-icon">
```

Comment:
- [ ] I'm getting this rails page error after running the local server 👇🏻 Please fix this 💯 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/227734538-10e54294-d7bb-465b-b19a-8a4ba929bd7b.png)


- [ ] Please make sure to update the remote deployment URL since all the style changes required by the first reviewer still not yet resolve.


- [ ] And remember to follow the provided design (width, font, size, height, background, color...)
 Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
* Make sure to use the same background color. 👍🏻 
*  Add this white border around the Add New Category button text. 👍🏻 
* Make sure you respect the dimension of the button (width, height, font size)...
5. The Transaction page and button have a similar problem 👇🏻 . Kindly fix it as well. 💯 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/227734938-c9a7d9d5-d5ae-47f9-93a9-e947f4caec9d.png)